pywren/pywren_ibm_cloud/action/jobrunner.py

```diff
@@ -135,17 +136,35 @@ def write_stat(stat, val):
     write_stat('data_download_time',
                data_download_time_t2-data_download_time_t1)
 
+    # Verify storage parameters - Create clients
+    func_sig = inspect.signature(loaded_func)
+    
+    if 'storage' in func_sig.parameters:
+        if 'ibm_cos' in storage_config:
+            func_storage = COSBackend(storage_config['ibm_cos'])
+        if 'swift' in storage_config:
+            func_storage = SwiftBackend(storage_config['swift'])
+        
+        loaded_data['storage'] = func_storage
+    
+    if 'ibm_cos' in func_sig.parameters:
+        func_storage = COSBackend(storage_config['ibm_cos'])
+        loaded_data['ibm_cos'] = func_storage
+    
+    if 'swift' in func_sig.parameters:
+        func_storage = SwiftBackend(storage_config['swift'])
+        loaded_data['swift'] = func_storage
+    
+    if 'internal_storage' in func_sig.parameters:
```

Comment:
@JosepSampe do we expect "internal_storage" to be in function parameters?

Reply:
@gilv reduce() method in executor.py needs internal_storage. Is for that I write it here.